Formula/neko.rb

```diff
@@ -33,6 +40,9 @@ class Neko < Formula
   depends_on "openssl"
 
   def install
+    # make has problem doing parallel build using nekoc/nekoml.
```

Comment:
Is there an issue for this? We consider `ENV.deparallelize` a bug.

Reply:
I'm not sure if it is make's fault or nekoc/nekoml's.

The root cause seems to be make firing up multiple builds of the same target in parallel. Since nekoc/nekoml simply writes to the output file, multiple concurrent writes will corrupt the output. I guess C compilers are fine because they are "smart" to write to a temp file first.

FYI, ninja builds neko HEAD fine in parallel since ninja wouldn't start redundant builds like make. We can update the brew formula to use ninja for the next neko version. Neko 2.1.0 still have some problem with using ninja.

Reply:
OK